JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1,12 +1,12 @@
-The DataGrid uses an integrated DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component to display predefined and custom commands. To add or remove toolbar items, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onToolbarPreparing) function. Use its parameter's **toolbarOptions** field to access the [Toolbar's configuration](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/) object. This object includes the command collection: the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array.
+DataGrid's toolbar allows you to customize predefined controls and add DevExtreme components and custom elements. To do this, use the [toolbar](Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/).[items[]](Documentation/21_2/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/items/) array. Declare the array elements in the order the controls should have in the UI.
```

Comment:
> toolbar allows you to customize predefined controls

These controls belong to what?

> add DevExtreme components and custom elements

Add where?

> [items[]](Documentation/21_2/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/items/)

Link contains version number.